src/constants/TopicTypes.js

```diff
@@ -16,6 +16,7 @@ export const TopicTypes = {
   collection: 'collection',
   class: 'class',
   container: 'container',
+  collectionGroup: 'collectionGroup',
```

Comment:
I dont see such type in the spec - https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json

Reply:
I think it's here :) 
https://github.com/apple/swift-docc/blob/a346e6999be87a69d579d2cb629056578ccd68cd/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json#L2115

Reply:
`topicTypes` are meant to represent the navigator topics. I understand the reasoning why we are adding the `collectionGroup`, but it just feels a bit odd. 

@mportiz08 what do you think?

Reply:
The reason why I added it here is because this is also where we compute color for the hero section. Is there a better way to implement this?

Reply:
Discussed with Dobri offline. Moving the color logic to `role.js`, since TopicTypes are meant for navigator and colors are not used there.